src/modules/AccountSettings/SettingOg/AdvanceMode.tsx

```diff
@@ -25,22 +25,25 @@ import { Textarea } from '@/components/ui/textarea'
 import { useToast } from '@/components/ui/use-toast'
 import { patchUpdateCustomOg, postAddNewCustomOg } from '@/lib/api'
 import { trackEvent } from '@/lib/firebase'
-import { deparse } from '@/lib/jsx-deparse'
-import JSXParser from '@/lib/jsx-parser'
+import { jsonToJsx, jsxToJson } from '@/lib/jsx-parser'
 import { CustomOg, UserProfile } from '@/lib/types'
 
 const schema = object({
-  publik: string('Kode laman publik perlu disi terlebih dahulu.', [
-    minLength(2, 'Kode laman publik butuh paling tidak 2 karakter.'),
-    maxLength(10000, 'Kode laman publik hanya bisa maksimal 10000 karakter.'),
-  ]),
-  question: string('Kode laman pertanyaan perlu disi terlebih dahulu.', [
-    minLength(2, 'Kode laman pertanyaan butuh paling tidak 2 karakter.'),
-    maxLength(
-      10000,
-      'Kode laman pertanyaan hanya bisa maksimal 10000 karakter.',
-    ),
-  ]),
+  publik: optional(
+    string('Kode laman publik perlu disi terlebih dahulu.', [
+      maxLength(10000, 'Kode laman publik hanya bisa maksimal 10000 karakter.'),
+    ]),
+    '',
+  ),
+  question: optional(
+    string('Kode laman pertanyaan perlu disi terlebih dahulu.', [
+      maxLength(
+        10000,
+        'Kode laman pertanyaan hanya bisa maksimal 10000 karakter.',
+      ),
+    ]),
+    '',
+  ),
```

Comment:
The fields are made optional in case the user wants to only customize one of these.